identity/src/main/java/com/stripe/android/identity/ui/ConsentScreen.kt

```diff
@@ -243,17 +237,35 @@ private fun SuccessUI(
 
         var acceptState by remember { mutableStateOf(LoadingButtonState.Idle) }
         var declineState by remember { mutableStateOf(LoadingButtonState.Idle) }
+        var scrolledToBottom by remember { mutableStateOf(false) }
 
-        LoadingButton(
-            modifier = Modifier
-                .padding(bottom = 10.dp)
-                .semantics { testTag = acceptButtonTag },
-            text = consentPage.acceptButtonText.uppercase(),
-            state = acceptState
-        ) {
-            acceptState = LoadingButtonState.Loading
-            declineState = LoadingButtonState.Disabled
-            onConsentAgreed(requireSelfie)
+        if (scrollState.value == scrollState.maxValue) {
+            scrolledToBottom = true
+        }
```

Comment:
This will run on every composition. I think the following would be better here. Can you give that a try?

```kotlin
val scrolledToBottom by remember(scrollState) {
  scrollState.value == scrollState.maxValue
}
```

Reply:
hmm I tried both 
```
val scrolledToBottom by remember(scrollState) {
            mutableStateOf(scrollState.value == scrollState.maxValue)
        }
```
and 

```
val scrolledToBottom = remember(scrollState) {
            scrollState.value == scrollState.maxValue
        }
```
But none of them works

Reply:
maybe [derivedStateOf](https://developer.android.com/jetpack/compose/side-effects#derivedstateof)?

Reply:
updated with with @tillh-stripe 's recommendation :)
```
var scrolledToButton by remember { mutableStateOf(false) }
        LaunchedEffect(scrollState.value) {
            if (!scrolledToButton) {
                scrolledToButton = scrollState.value == scrollState.maxValue
            }
        }
```